content/get-involved/report-issues/how-issues-are-sorted.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How issues are sorted
+---
+
+# How issues are sorted
+
+## What happens to the issue you have reported
+
+PrestaShop Quality Assurance team (aka QA) uses a transparent definition for the criteria used to qualify issues' severity, and how they should be applied through labels on github issues.
+
+Please note that severity is to be distinguished from priority. Indeed, severity is used to measure the negative impact that a bug has on a system, a feature, a component or on the project development. It is usually defined by the QA team. As for the priority, it is used to organize all the tasks (bugs, improvements, features, technical tasks) that have to be done in order to meet the project's deadlines. 
```

Comment:
I think there's a lack of details regarding the priority. We don't clearly see a priority for issues.

```suggestion
Please note that severity is to be distinguished from priority. Indeed, severity is used to measure the negative impact that a bug has on a system, a feature, a component or on the project development. It is usually defined by the QA team. As for priority, it is used by product managers to organize all the tasks (bugs, improvements, features, technical tasks) that have to be done in order to meet the project's deadlines. You can see priorities on Kanban related to the issue.
```

Reply:
+ add a link to the kanban ?

Reply:
Priority is for teams, issues are not prioritized from the community's standpoint.